components/normalizer/src/lib.rs

```diff
@@ -549,7 +563,15 @@ where
         decompositions: &'data DecompositionDataV1,
         tables: &'data DecompositionTablesV1,
     ) -> Self {
-        Self::new_with_supplements(delegate, decompositions, None, tables, None, 0xC0)
+        Self::new_with_supplements(
+            delegate,
+            decompositions,
+            None,
+            tables,
+            None,
+            0xC0,
+            IgnorableBehavior::Unsupported,
```

Comment:
The underlying struct can be used for non-UTS46 purposes: the usual normalizations. For those, we expose contiguous-buffer APIs, and the contiguous-buffer versions currently can't deal with ignorables, and since there is currently no use case for them to deal with ignorables, I think it's better to have a state that they can `debug_assert!` against than to add support for ignorables is the contiguous-buffer entry points.